### DISCUSSION TITLE
Take codebase spoken language into account

### DISCUSSION THREAD

**Discussion by jebarjonet** [community/community#59632] [Copilot Conversations]
### Select Topic Area

Product Feedback

### Body

This feedback is about Copilot Nightly suggesting code updates, please let me know if this should be posted elsewhere.

I am working in a French codebase where all comments and docs are written in French. When Copilot suggests code updates and adds comments, they are in English, but I expect them to be in French using the same way of speaking as the other comments.

I can specify "in French" at the end of my command, which works well, but I think it would be even better if it did it automatically.

**Comment by Akash1134** [comment]
Hi @jebarjonet ,

Thank you for sharing the feedback. We will pass it on to the concerned team. 🙂
